src/lua/compat.lua

```diff
@@ -15,14 +14,36 @@ local options_format = {
     action         = 'function/nil',
 }
 
+local JSON_ESCAPE_BRIEF = [[
+Whether to escape the forward slash symbol '/' using a backslash in a
+json.encode() result. The old and the new behavior produce a result, which
+is compatible ith the JSON specification. However most of other JSON encoders
+don't escape the forward slash, so the new behavior is considered more safe.
+
+https://github.com/tarantool/tarantool/wiki/compat%3Ajson_escape_forward_slash
```

Comment:
It's quite strange for me to see the URL in the code. Do we really need it here?

Reply:
We do, it is an agreed part of brief. We need to give user an easy way to access option wiki.